src/components/styles.scss

```diff
@@ -11,3 +11,39 @@
     }
   }
 }
+
+// CookieConsent component
+.cookieConsent {
```

Comment:
Looks good on desktop! On mobile layout the consent text is not fully visible...